### PR TITLE
Fix height to big for iPhone when using many apps

### PR DESCRIPTION
**Pull request by kevin147147**

### CODE REVIEW

**Comment by kevin147147** [comment]
@MorrisJobke

**Comment by MorrisJobke** [comment]
cc @nextcloud/designers

**Comment by MorrisJobke** [comment]
@kevin147147 Could you explain the fix and also explain the issue? best with screenshots of before and after?

**Comment by kevin147147** [comment]
Before

e8dd6d23-f036-4683-8ee9-8316e767f23d


After
4bad7297-2b03-4abb-b2dd-584b568d2f18

**Comment by kevin147147** [comment]
Look at the bottom of the screenshots. The Dropdown-Menu is now fully visible. Before you could only barely touch the last entry.

It has to do with using the viewport height:
https://nicolas-hoizey.com/2015/02/viewport-height-is-taller-than-the-visible-part-of-the-document-in-some-mobile-browsers.html

File: core/css/header.scss [hunk 1]
```diff
@@ -324,7 +324,7 @@ nav[role='navigation'] {
 /* Apps management */
 
 #apps {
-	max-height: calc(100vh - 100px);
+	max-height: calc(100vh - 200px);
```
Comment (MorrisJobke) [inline core/css/header.scss]: Could you change this to `inherit` then it looks perfect ;) We just tested it over here and then it looks how it is supposed to.

**Comment by MorrisJobke** [comment]
While doing this, amend the commit and add a sign-off message to the commit ;) See https://github.com/nextcloud/server/blob/master/CONTRIBUTING.md#sign-your-work for more details

**Comment by kevin147147** [comment]
max-height: inherit doesn't do it for me. It then has the same height as before and the menu gets under the safari bar again like before. How did you test it? With a real iOS device? Sadly the responsive design modes in chrome, etc. will not show this bug.

**Comment by jancborchardt** [comment]
On small screens this would cut off too much? If it only is an issue in Safari mobile, can it be solved differently? (Like max-height: 80% or something else.)

**Comment by skjnldsv** [comment]
You can also set it to 80vh yes :)
Just put `max-height: unset` to `.header-left #navigation` also then

**Comment by kevin147147** [comment]
This is how it looks even on horizontal. What smaller devices are you thinking about?

62e531d9-c62c-43d7-991e-0cd288882c4c

**Comment by kevin147147** [comment]
Well 80vh just ever so slightly does it in vertical. In horizontal it looks nearly like above :)

f65e1833-f0f0-4cbd-8cb2-1b2c7c3008d6

**Comment by MorrisJobke** [comment]
Let's wait here for #9982 as well, because it fixes the issue on iOS that the control bar of the browser is always there.

**Comment by ChristophWurst** [comment]
> Let's wait here for #9982 as well, because it fixes the issue on iOS that the control bar of the browser is always there.

That PR was merged. @kevin147147 could you please re-verify that the patch is still necessary? There have been lots of changes and fixes for Nextcloud 14 with the apps menu and there's the possibility that your issue has been fixed by that. Thanks!

**Comment by kevin147147** [comment]
That works

**Comment by ChristophWurst** [comment]
Perfect. Thank you for your feedback!

**Pull request closed without merging**